### DISCUSSION TITLE
Any idea about implementation for multiple (sub)domain authentication with supabase Auth ??

### DISCUSSION THREAD

**Discussion by rahul3v** [supabase/supabase#6300] [Questions]
How can I enable authentication for multiple sub-domains if a user authenticated from one of any domain....!!

@kiwicopple any comments...!!

**Comment by libovness** [comment]
@rahul3v did you figure this out?

**Reply by rahul3v** [reply]
nope @libovness, not yet !!

**Comment by trevorpfiz** [comment]
#5742
